src/rez/rezconfig.py

```diff
@@ -534,6 +534,19 @@
     "*PATH"
 ]
 
+# Some shells may require multiple types of pathing, so this option provides
+# a way to define variables on a per-shell basis to convert for shell pathing
+# instead of the pathing provided above or no modification at all.
+shell_pathed_env_vars = {
+    "gitbash": ["PYTHONPATH"]
+}
+
+# If set to True, completely disables any path transformations that would occur
+# as a result of both the shell and the settings in "pathed_env_vars" and
+# "shell_pathed_env_vars". This is meant to aid in debugging and should be
+# False unless needed.
+disable_normalization = False
```

Comment:
From what I remember, we initially said we want it to be off by default. So I'd rename it `enable_path_normalization` and set the value to `False`.